docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Enroll an agent into automatic updates
+description: How to enroll an agent into automatic updates
+---
+
+<Admonition type="warning">
+The instructions vary between Teleport Cloud users and self-hosted
+Teleport users. Make sure to select the right scope (OpenSource, Enterprise, or
+Cloud) in the scope selector at the top of the page.
+</Admonition>
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  Automatic agent update is available starting from Teleport `13.0`.
+</Details>
+
+Starting with Teleport 13, agents running in systemd or Kubernetes environments
+can be automatically updated. The [automatic updates architecture
+page](../../architecture/automatic-agent-update.mdx) describes how agent
+updating works.
+
+This guide explains how to enroll an existing Teleport agent into automatic
+updates.
+
+## Requirements
+
+- a Teleport agent either:
+  - started via systemd
+  - deployed with the `teleport-kube-agent` Helm chart
+<ScopedBlock scope={["oss","enterprise"]}>
+- automatic update infrastructure set up. For Self-Hosted users this means you
+  already followed [this guide](./self-hosted-automatic-agent-updates.mdx) and
+  know your version server URL and release channel
+</ScopedBlock>
+<ScopedBlock scope="cloud">
+- as a Teleport Cloud user, you must check if your Could Tenant is enrolled
+  into automatic updates.
+</ScopedBlock>
+
+## Enroll instructions
+
+<Tabs>
+<TabItem label="systemd">
+
+<ScopedBlock scope={["oss","enterprise"]}>
+- create the upgrade configuration directory
```

Comment:
I don't think these steps should be bulleted. Each step should be sentence-cased, and explanations before a command should end with `:`. For example:

```suggestion
Create the upgrade configuration directory:
```

Reply:
Re-resolving because I fixed it while making other edits.